### PR TITLE
Fix/issue 6195 classic block shortcuts visible tooltips

### LINKED ISSUE #6195: Classic Block: Consider to make the editor keyboard shortcuts visible in the tooltips
In WordPress, https://core.trac.wordpress.org/changeset/38897 made some editor keyboard shortcuts more discoverable by adding them to the buttons tooltips. See also https://core.trac.wordpress.org/ticket/43773

screen shot 2018-04-16 at 10 19 59

Instead, in Gutenberg the keyboard shortcuts are not exposed in the tooltips, not even the ones in the Classic editor (which is TinyMCE):

screen shot 2018-04-16 at 10 33 09

screen shot 2018-04-16 at 10 33 19

For more details on how the shortcuts are added and translated, see `src/wp-includes/js/tinymce/plugins/wordpress/plugin.js` in WordPress, which gets the translations from PHP via the localized editor settings object.

Aside:
notice the positioning (and size) of the tooltips in the classic editor block (TinyMCE) is different from the one in the Gutenberg blocks.

### PR DESCRIPTION
**Pull request by Firestorm980**
## Description
Fixes an issue where tooltips on the classic editor no longer show keyboard shortcuts. 
Fixes https://github.com/WordPress/gutenberg/issues/6195

## How has this been tested?
- Ran tests from the Gutenberg docker environment `npm run test` & `npm run test-e2e`
- Added all different content into the editable area to make sure it still works as expected

## Types of changes
- Moving initialization to when the editor is focused vs on window DOM content loaded
- Added `onFocus` handler for the editing area
- Added `dangerouslySetInnerHTML` on editing area so content is visible on page load.

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation.

### CODE REVIEW

File: packages/block-library/src/classic/edit.js [hunk 1]
```diff
@@ -198,6 +202,9 @@ export default class ClassicEdit extends Component {
 				key="editor"
 				id={ `editor-${ clientId }` }
 				className="wp-block-freeform block-library-rich-text__tinymce"
+				contentEditable="true"
+				dangerouslySetInnerHTML={ { __html: this.props.attributes.content } }
```
Comment (tofumatt) [inline packages/block-library/src/classic/edit.js]: Why are we using `dangerouslySetInnerHTML` here? I don't see the content we're loading in here being sanitised anywhere here so I'm not sure it's safe to load. Even if it's safe upstream I would want to see it sanitised in this file for clarity's sake and I'd want to see tests making sure there is no potential for code injection here.

File: packages/block-library/src/classic/edit.js [hunk 2]
```diff
@@ -64,6 +62,12 @@ export default class ClassicEdit extends Component {
 	initialize() {
 		const { clientId } = this.props;
 		const { settings } = window.wpEditorL10n.tinymce;
+
+		// Prevent multiple inits on the same classic block
+		if ( this.editor && this.editor.initialized ) {
```
Comment (tofumatt) [inline packages/block-library/src/classic/edit.js]: Why are we calling `initialize` again in the first place on an already initialised block?

**Review by tofumatt — CHANGES_REQUESTED** [review]
The use of `dangerouslySetInnerHTML` here has me worried; could you clarify the intent around using it and make sure it's being used safely with some tests?

**Comment by tofumatt** [comment]
Sorry this one took forever to get to; I've been a bit behind on my review queue. Thanks a million for your patience, really sorry again.

File: packages/block-library/src/classic/edit.js (hunk 2 above)
Comment (Firestorm980) [reply packages/block-library/src/classic/edit.js]: @tofumatt initialize being called on page load was the main issue, so we need to initialize them in a more "on demand" fashion to get around it. Perhaps an alternate here is to disable the `onFocus` handler after the block has been initialized so we wouldn't need this check?

**Comment by timwright12** [comment]
@gziolo Do you have any thoughts  on this one?

**Comment by aduth** [comment]
Per https://github.com/WordPress/gutenberg/issues/6195#issuecomment-477928992 , this appears to have been separately fixed upstream in the WordPress repository. Thanks all the same for your proposed contribution here!

See also:

- https://core.trac.wordpress.org/ticket/35710
- https://core.trac.wordpress.org/changeset/45066

**Pull request closed without merging**